packages/ember-cli-fastboot/test/fastboot-location-config-test.js

```diff
@@ -16,15 +16,15 @@ describe('FastBootLocation Configuration', function () {
 
     return app
       .create('fastboot-location-config', {
-        emberVersion: 'latest',
-        emberDataVersion: 'latest',
+        emberVersion: '~3.28.12',
```

Comment:
@Gaurav0, thanks heaps for doing this. 

Why fix the version to 3.?

Reply:
It matches the default version of the ember-cli-fastboot addon being used in the test.

In a followup PR, we should probably update all the defaults to 4.12. Right now, just want to get things working again.